### ISSUE TITLE
porter installation runs list doesn't populate parameter values

### ISSUE THREAD

**Issue by carolynvs** [getporter/porter#2199]
## Describe the bug
The `porter installation runs list` command has a parameters field but all of the parameters are set to "".

## To Reproduce
1. `porter install -r ghcr.io/getporter/examples/porter-hello:v0.2.0 --param name=myname --debug`
2. `porter installation runs list examples/porter-hello --output yaml`

Note that in the output of the runs list command, the name parameter is empty.

## Expected behavior
The name parameter should either display the actual resolved parameters, or should nil out the parameters field so that it's not misleading.

## Porter Command and Output
```console
$ porter installation runs list examples/porter-hello -o yaml
- id: 01G6KC06BF9W46QQFKSR925TA1
  bundle: ghcr.io/getporter/examples/porter-hello:v0.2.0
  version: 0.2.0
  action: install
  parameters:
    name: ""
    porter-debug: ""
    porter-state: null
  started: 2022-06-27T15:00:16.241363-05:00
  stopped: 2022-06-27T15:00:17.438817-05:00
  status: succeeded
```

## Version
1.0.0-alpha.22

**Comment by joshuabezaleel** [comment]
I tried to print the `r.Parameters.Parameters` on this particular line https://github.com/getporter/porter/blob/release/v1/pkg/storage/run.go#L167 which I noticed that the value that we want to print is in the `param.Source.Value` while all of the `param.Value` are empty. 

![image](https://user-images.githubusercontent.com/7043511/176323186-772264ee-bbb9-4bef-bbf3-8735e8fdf1ec.png)

The solution for the value to get printed might be as just to change the input on this one (https://github.com/getporter/porter/blob/release/v1/pkg/storage/run.go#L168) from `param.Value` to using `param.Source.Value` but I just want to confirm since I have a hunch that it was not expected and we want `param.Value` to get populated by the value on `param.Source.Value` instead. 

![image](https://user-images.githubusercontent.com/7043511/176323280-8614d497-47a8-4a99-acc5-c55c9311b8a9.png)

**Comment by joshuabezaleel** [comment]
I assumed the latter one since I was looking at these commentaries. https://github.com/getporter/porter/blob/release/v1/pkg/secrets/strategy.go#L46-L58

**Comment by carolynvs** [comment]
Yeah this is why I'm recommending that we nil out this field. It requires multiple db and service calls to resolve the values  properly because it could be a secret, or the contents of a file, etc. The parameter values are not needed for the list command, and it's just confusing that it's half populated.
